tests/hijack/async.js

```diff
@@ -81,6 +84,9 @@ Tinytest.add(
       return acc;
     }, {});
 
+    console.log('compute', events.compute);
+    console.log('async', events.async);
+
     test.isTrue(events.compute > 0);
```

Comment:
It might be good to change this to > 99 to ensure it is including the compute time from the while loop, and not just other random compute time in the method.

A more accurate check might be to create a mongo cursor (`Collection.find()`) in the method, and make sure that there is a top-level db event in the trace (if the async event didn't end, then the db event would be nested).